cloudbuild-3_11.yaml

```diff
@@ -0,0 +1,60 @@
+steps:
+  - id: 'pull image'
+    name: 'gcr.io/stargateio/stargate-builder:v1.0.3'
+    entrypoint: 'bash'
+    args: [ '-c', 'echo pulled builder' ]
+  - id: 'fetch_secret'
+    waitFor: [ '-' ]
+    name: gcr.io/cloud-builders/gcloud
+    entrypoint: 'bash'
+    args: [ '-c', "gcloud secrets versions access latest --secret=stargate-artifactory --format='get(payload.data)' | tr '_-' '/+' | base64 -d > /workspace/ci/custom-settings.xml" ]
+  - id: 'fetch codacy secret'
+    waitFor: [ '-' ]
+    name: gcr.io/cloud-builders/gcloud
+    entrypoint: 'bash'
+    args: [ '-c', "gcloud secrets versions access latest --secret=codacy-project-token --format='get(payload.data)' | tr '_-' '/+' | base64 -d > /workspace/ci/codacy-project-token" ]
+  - id: 'init_cache'
+    name: 'gcr.io/cloud-builders/gsutil'
+    waitFor: [ '-' ]
+    args:
+      - '-m'
+      - 'rsync'
+      - '-r'
+      - '-x'
+      - '^.*io/stargate.*$'
+      - 'gs://${_BUCKET}/cache/.m2'
+      - '/cache/.m2'
+    volumes:
+      - path: '/cache/.m2'
+        name: 'm2_cache'
+  - id: 'cassandra-3.11'
+    waitFor: [ 'fetch_secret', 'init_cache' ]
+    name: 'gcr.io/stargateio/stargate-builder:v1.0.3'
+    entrypoint: 'bash'
+    args: [ 'ci/test.sh' ]
+    volumes:
+      - path: '/cache/.m2'
+        name: 'm2_cache'
+    env:
+      - 'MAVEN_OPTS=-Dmaven.repo.local=/cache/.m2'
+      - 'PERSISTENCE_BACKEND=cassandra-3.11'
+      - 'COMMIT_ID=$COMMIT_SHA'
+  - name: 'gcr.io/cloud-builders/gsutil'
+    waitFor: [ 'cassandra-3.11' ]
+    args:
+      - '-m'
+      - 'rsync'
+      - '-r'
+      - '-x'
+      - '^.*io/stargate.*$'
+      - '/cache/.m2'
+      - 'gs://${_BUCKET}/cache/.m2/'
+    volumes:
+      - path: '/cache/.m2'
+        name: 'm2_cache'
+
+substitutions:
+  _BUCKET: 'stargateio-github_cloudbuild'
+timeout: 14400s
+options:
+  machineType: 'N1_HIGHCPU_8'
```

Comment:
4 might be enough. In my local runs I do not see high CPU usage, the run time mostly comes from restart and schema delays, I think.... Just an observation.

Reply:
I was thinking the same thing but the only options are 8 and 32.